feature_layers/display-subtype-feature-layer/src/main/java/com/esri/samples/display_subtype_feature_layer/DisplaySubtypeFeatureLayerController.java

```diff
@@ -61,18 +63,23 @@ public void initialize() {
       // display the current map scale 
       mapView.addMapScaleChangedListener(mapScaleChangedEvent ->
         currentMapScaleLabel.setText("Current map scale: 1:" + Math.round(mapView.getMapScale())));
-      
+
       // set the viewpoint to Naperville, Illinois
       Viewpoint initialViewpoint = new Viewpoint(new Envelope(-9812691.11079696, 5128687.20710657,
         -9812377.9447607, 5128865.36767282, SpatialReferences.getWebMercator()));
       map.setInitialViewpoint(initialViewpoint);
 
       // create a subtype feature layer from the service feature table, and add it to the map
       ServiceFeatureTable serviceFeatureTable = new ServiceFeatureTable("https://sampleserver7.arcgisonline" +
-        ".com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/100");
+        ".com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/0");
+
+      // set user credentials to authenticate with the service
+      UserCredential userCredential = new UserCredential("viewer01", "I68VGU^nMurF");
```

Comment:
Same question about this PW - it looks quite realistic, so not sure if it's supposed to be here? If the PW is public, then there's no need in creating a 'secure' password, `viewer01.password` would work just as well and would be memorable if a user tries to play with authentication forms?

There's possibly a few more cases like this in the remainder of the files, so I won't comment on those